pkg/apis/stork/v1alpha1/migration.go

```diff
@@ -29,6 +29,7 @@ type MigrationSpec struct {
 	IncludeOptionalResourceTypes []string          `json:"includeOptionalResourceTypes"`
 	SkipDeletedNamespaces        *bool             `json:"skipDeletedNamespaces"`
 	TransformSpecs               []string          `json:"transformSpecs"`
+	IgnoreOwnerReferences        *bool             `json:"ignoreOwnerReferences"`
```

Comment:
It can get a a bit misleading I think.
I know the intent is to ignore owner reference fields during migration.
However someone might read it like, ignore resources from migration which have owner reference set. 
I think it might be a bette to use IgnoreOwnerReferencesCheck or IncludeResourcesWithOwnerReference

Reply:
Changed it to IgnoreOwnerReferencesCheck